squash_wip.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+type Replacer func(string) string
+
+func squashWip(configuration Configuration) {
+	if endsWithWipCommit(configuration) {
+		sayInfo("Make sure the final commit is a manual commit before squashing")
+		exit(1)
+		return
+	}
+
+	currentBaseBranch, currentWipBranch := determineBranches(gitCurrentBranch(), gitBranches(), configuration)
+	if gitCurrentBranch() != currentWipBranch {
+		sayInfo("Make sure you are on the wip-branch before running quash-wip")
```

Comment:
In this case, one isn't mob programming. Try to use the same logic as in other methods when one should be mob programming but isn't.

Reply:
good catch, did that.